packages/babel-core/src/gensync-utils/fs.ts

```diff
@@ -10,14 +10,8 @@ export const readFile = gensync<(filepath: string, encoding: "utf8") => string>(
 
 export const exists = gensync<(filepath: string) => boolean>({
```

Comment:
We can remove this function, and use `fs.existsSync` directly in https://github.com/babel/babel/blob/fb908fd87e1bf8a5ebe2c881c0baab1488c917e7/packages/babel-core/src/config/files/configuration.ts#L43.

`findConfigUpwards` than doesn't need to be a generator:
- We also need to make it a normal function in packages/babel-core/src/config/files/index-browser.ts
- We can update its usage in `resolveRootMode` in `packages/babel-core/src/config/partial.ts`, which doesn't need to be a generator anymore

Reply:
Wow, I'm so brainwashed by work, we'd never bother to push de-asyncing out like this. :smile: